source/scss/components/_colors.scss

```diff
@@ -0,0 +1,140 @@
+.bg-orange {
```

Comment:
I think it is not necessary to create a component file to add classes. We are using the SCSS variables and not the classes.

Reply:
If you like to create CSS for showing in pattern lab, which is not for the production code, you can use source/css/pattern-scaffolding.css. Do you think, we should include it in the Readme?